customer/src/main/java/no/unit/nva/customer/update/UpdateCustomerHandler.java

```diff
@@ -18,65 +18,31 @@
 import nva.commons.core.JacocoGenerated;
 import org.apache.http.HttpStatus;
 
-import java.util.List;
-import java.util.UUID;
-
 public class UpdateCustomerHandler extends ApiGatewayHandler<CustomerDto, CustomerDto> {
 
-    public static final String ID_NAMESPACE_ENV = "ID_NAMESPACE";
     public static final String IDENTIFIER = "identifier";
     public static final String IDENTIFIER_IS_NOT_A_VALID_UUID = "Identifier is not a valid UUID: ";
-
-    private final CustomerMapper customerMapper;
     private final CustomerService customerService;
 
     /**
      * Default Constructor for UpdateCustomerHandler.
      */
     @JacocoGenerated
     public UpdateCustomerHandler() {
-        this(defaultCustomerService(),
-            defaultCustomerMapper(),
-            new Environment()
-        );
-    }
-
-    @JacocoGenerated
-    private static DynamoDBCustomerService defaultCustomerService() {
-        return new DynamoDBCustomerService(
-            AmazonDynamoDBClientBuilder.defaultClient(),
-            ObjectMapperConfig.objectMapper,
-            new Environment());
-    }
-
-    @JacocoGenerated
-    private static CustomerMapper defaultCustomerMapper() {
-        String namespace = new Environment().readEnv(ID_NAMESPACE_ENV);
-        return new CustomerMapper(namespace);
+        this(defaultCustomerService(), new Environment());
```

Comment:
All handlers changed in very similar ways, due to their common structure. The changes are almost identical in every handler